google/cloud/gkeconnect/quickstart/quickstart.cc

```diff
@@ -14,21 +14,27 @@
 
 //! [all]
 #include "google/cloud/gkeconnect/gateway/v1/gateway_control_client.h"
+#include "google/cloud/internal/absl_str_cat_quiet.h"
```

Comment:
We don't want customers including internal headers in their applications, so we shouldn't include internal headers in our examples/quickstarts.

Reply:
It is interesting, I am using `#include "absl/strings/str_cat.h"`, it is automatically replace by `checkers-pr`.

Reply:
https://screenshot.googleplex.com/AVbJ46XqY6yxaPG
Does that mean we can't use `absl::StrCat` in quickstart code?
I can replace it with plain string addition.

Reply:
Yeah. I think we tend to do string addition in these.